frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx

```diff
@@ -472,7 +472,8 @@ export const OperandDetails = connectToModel(({ crd, csv, kindObj, obj }: Operan
   const { kind, status } = obj;
   const [errorMessage, setErrorMessage] = React.useState(null);
   const handleError = (err: Error) => setErrorMessage(err.message);
-  const schema = crd?.spec?.validation?.openAPIV3Schema ?? (definitionFor(kindObj) as JSONSchema6);
+  const schema =
+    crd?.spec?.versions[0]?.schema?.openAPIV3Schema ?? (definitionFor(kindObj) as JSONSchema6);
```

Comment:
I'd think we need to search for the specific version we need here, @TheRealJon ?

Reply:
Yeah, @rhamilto We already have logic that finds the correct owned/required CRD in the csv, so we just need to destructure the `version` field on L479 as well. Then you can move this declaration below that one, and use `version` to find the appropriate schema in `crd.spec.versions`.